panopticum/management/commands/importer.py

```diff
@@ -0,0 +1,1085 @@
+import csv
+import logging
+import re
+import sys
+import json
+import traceback
+
+from django.core.exceptions import MultipleObjectsReturned
+from django.core.management.base import BaseCommand, CommandError
+from django.core import serializers
+
+from panopticum.models import *
+
+
+# Provides all the utility functions either in static form or as part of instance methods.
+class Utility:
+    comp_categories = \
+        {'Category1': {'description': '', 'order': 1}}

```

Comment:
Why we need to hardcode the data we have in config or csv files?

Reply:
Initially it all started to have these structures with default values in code and then made it part of defaults.json. I kept them in code for users to see what structure each variable follows.

defaults.json will always overwrite the code defaults.

Reply:
Let's initial data to (init.json)[https://github.com/perfguru87/panopticum/blob/master/panopticum/fixtures/init.json]